test/integration/functional_test.go

```diff
@@ -637,7 +637,7 @@ func validateCacheCmd(ctx context.Context, t *testing.T, profile string) {
 
 	t.Run("cache", func(t *testing.T) {
 		t.Run("add_remote", func(t *testing.T) {
-			for _, img := range []string{"k8s.gcr.io/pause:3.1", "k8s.gcr.io/pause:3.3", "k8s.gcr.io/pause:latest"} {
+			for _, img := range []string{"k8s.gcr.io/pause:3.1", "k8s.gcr.io/pause:3.3", "k8s.gcr.io/pause:3.4.1"} {
```

Comment:
testing latest was one of the test cases, lets keep that, minikbue cache add should still be tested against :latest